packages/connection/src/class-manager.php

```diff
@@ -26,7 +26,23 @@ class Manager {
 	const SECRETS_EXPIRED        = 'secrets_expired';
 	const SECRETS_OPTION_NAME    = 'jetpack_secrets';
 	const MAGIC_NORMAL_TOKEN_KEY = ';normal;';
-	const JETPACK_MASTER_USER    = true;
+
+	/**
+	 * Constant used to fetch the master user token. Deprecated.
+	 *
+	 * @deprecated 9.0.0
```

Comment:
Should the deprecated version by based on the package ver, not the plugin ver?

Reply:
> Is there a better term than MASTER_USER?

Primary user. Connection owner. I'm total in favor of changing the language there, but it's really hard to do it little by little because you end up with very inconsistent code. For example, this option is stored in the database as `master_user`. It feels like this is best done in one move that changes everything while keeping things consistent.

Reply:
> Should the deprecated version by based on the package ver, not the plugin ver?

That's an excellent question! It came up in our call last week and we talked about that - that deprecations inside packages should relate to the package version. How are we versioning packages?

Reply:
> deprecations inside packages should relate to the package version. How are we versioning packages?

Yeah, I think that would make sense. However we do not currently store a package's version inside the package itself, in a way that would make that number easily accessible in PHP.
We could start doing that, but that would require updating those numbers in the monorepo via a PR every time we want to update package versions in the mirrored repos.

This may require a longer discussion on a P2. :)

Reply:
Agree we have to discuss this. Not sure if this is a blocker for this PR, since we are currently deprecating things on packages based on Jetpack version (and also with the string "Jetpack" in the deprecation message, which is even worse). So I'd say we move on with this and open this discussion.